config/bootenv/rockpis-dev.txt

```diff
@@ -0,0 +1,3 @@
+verbosity=7
+extraargs=swiotlb=1024
+fdtfile='rockchip/rk3308-rock-pi-s.dtb'
```

Comment:
board dt is not stored in u-boot environment? fdtfile has no default value?

Reply:
The u-boot env uses the legacy name of `rockpi-s-linux`. 
I initially created a patch for this - but then found this to work as well. 
```patch
diff --git a/configs/rockpi-s-rk3308_defconfig b/configs/rockpi-s-rk3308_defconfig
index c0cb439..f9d68f0 100644
--- a/configs/rockpi-s-rk3308_defconfig
+++ b/configs/rockpi-s-rk3308_defconfig
@@ -9,7 +9,7 @@ CONFIG_ROCKCHIP_VENDOR_PARTITION=y
 CONFIG_TARGET_EVB_RK3308=y
 CONFIG_TARGET_ROCKPIS_RK3308=y
 CONFIG_SPL_STACK_R_ADDR=0xc00000
-CONFIG_DEFAULT_DEVICE_TREE="rockpi-s-linux"
+CONFIG_DEFAULT_DEVICE_TREE="rk3308-rock-pi-s"
 CONFIG_DEBUG_UART=y
 CONFIG_ANDROID_BOOT_IMAGE=y
 CONFIG_FIT=y
```

Reply:
@ashthespy I will fix it in the u-boot source to use the `rk3308-rock-pi-s` name for both legacy and dev.
We can then get rid of both `config/bootenv/rockpis-dev.txt` and `config/bootscripts/boot-rockpis-dev.cmd`.

I will give you a shout when it is done.

Reply:
@ashthespy Done with those two commits: https://github.com/piter75/rockchip-u-boot/commit/cde3eb3c6ece39d3d3e69d57ecfa905a59fc6330 https://github.com/armbian/build/commit/992d0aa88cb2cacaf16660ce153dc51b7817b0c4
Now Rock Pi S' u-boot is expecting `/boot/dtb/rockchip/rk3308-rock-pi-s.dtb` file.

Reply:
All branches now use the same bootenv and bootscript, thanks!

Reply:
> @ashthespy Done with those two commits: [piter75/rockchip-u-boot@cde3eb3](https://github.com/piter75/rockchip-u-boot/commit/cde3eb3c6ece39d3d3e69d57ecfa905a59fc6330) [992d0aa](https://github.com/armbian/build/commit/992d0aa88cb2cacaf16660ce153dc51b7817b0c4)
> Now Rock Pi S' u-boot is expecting `/boot/dtb/rockchip/rk3308-rock-pi-s.dtb` file.

Just a heads up  [piter75/rockchip-u-boot@cde3eb3](https://github.com/piter75/rockchip-u-boot/commit/cde3eb3c6ece39d3d3e69d57ecfa905a59fc6330)  doesn't seem to work for me, u-boot still looks for `rockpi-s-linux.dtb` Or am I being daft?

Reply:
> doesn't seem to work for me, u-boot still looks for `rockpi-s-linux.dtb` Or am I being daft?

It's not you. It is me ;p
I applied my knowledge of mainline u-boot to BSP one which behaves differently in this regard.
I fixed it in my BSP u-boot fork. It works well now.

Reply:
Prefect, thanks! :-) 

Side note: I tried playing with mainline u-boot, since there is support for rk3308 already, but wasn't able to get it to boot from my SD card, gotta dive back into that soon!